src/wp-includes/class-wp-scripts.php

```diff
@@ -871,8 +883,22 @@ private function get_dependents( $handle ) {
 	 */
 	private function get_intended_strategy( $handle ) {
 		$script_args = $this->get_data( $handle, 'script_args' );
+		$strategy    = isset( $script_args['strategy'] ) ? $script_args['strategy'] : false;
 
-		return isset( $script_args['strategy'] ) ? $script_args['strategy'] : false;
+		if ( $strategy && ! $this->is_valid_strategy( $strategy ) ) {
+			_doing_it_wrong(
+				__METHOD__,
+				sprintf(
+					/* translators: 1: $strategy, 2: $handle */
+					__( 'Invalid strategy `%1$s` set for `%2$s`, during script registration.' ),
```

Comment:
```suggestion
					__( 'Invalid strategy `%1$s` defined for `%2$s` during script registration.' ),
```

No need for the comma, it adds an incorrect pause :)